contracts/modules/TransferManager/VolumeRestrictionTM.sol

```diff
@@ -143,8 +143,8 @@ contract VolumeRestrictionTM is VolumeRestrictionTMStorage, ITransferManager {
             exemptIndex[_wallet] = exemptAddresses.length;
             exemptList[_wallet] = true;
         } else {
-            exemptAddresses[exemptIndex[_wallet]] = exemptAddresses[exemptAddresses.length -1];
-            exemptIndex[exemptAddresses[exemptIndex[_wallet]]] = exemptIndex[_wallet];
+            exemptAddresses[exemptIndex[_wallet] - 1] = exemptAddresses[exemptAddresses.length -1];
```

Comment:
How come we do this rather than using:
`exemptIndex[_wallet] = exemptAddresses.length - 1;`
above? (so the indexes aren't off by one)?

Reply:
Indexes are off by one always. While adding exemptAddresses, the address is pushed first and then the length of the array is used as the index.

Basically, the index we are using starts from 1 instead of 0.